.github/workflows/build-ui.yml

```diff
@@ -0,0 +1,66 @@
+name: Build UI
+
+on:
+  pull_request:
+    branches: [main]
+
+permissions:
+  contents: write
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4.1.1
+        with:
+          ref: ${{ github.head_ref }}
+          fetch-depth: 2
+
+      - id: check_label
+        name: Check PL labels
+        uses: docker://agilepathway/pull-request-label-checker:latest
+        with:
+          one_of: dependencies
+          repo_token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
We can add the label check at the job level, by something like:

```
if: ${{ github.event.label.name == 'dependencies' }}
```

Reply:
I am not sure about passing the github token to a docker image with `latest` tag, due to security concerns.

Reply:
Updated with pinned version